dali-mqtt-daemon.py

```diff
@@ -38,18 +44,40 @@
 
 HA_DISCOVERY_PREFIX="{}/light/dali2mqtt_{}/config"
 
-def gen_ha_config(light):
+class ConfigFileSystemEventHandler(FileSystemEventHandler):
+    def __init__(self, mqqt_client, config, config_file_name, driver_object, loop):
+        super().__init__()
+        self.mqqt_client = mqqt_client
+        self.config_file_name = config_file_name
+        self.driver_object = driver_object
+        self.config = config
+        self.loop = loop
+    
+    def on_modified(self, event):
+        logger.info("Detected changes in configuration file {}, reloading".format(event.src_path))
+        self.config = load_config_file(self.config_file_name)
+        self.mqqt_client.loop_stop()
+        self.mqqt_client = create_mqtt_client(self.driver_object, self.config["dali_lamps"], self.config["mqtt_server"], self.config["mqtt_port"], self.config["mqtt_base_topic"], self.config["ha_discover_prefix"])
+        self.mqqt_client.loop_start()
```

Comment:
This logic (Mqtt restart) should not belong to the FileSystemEventHandler, it should come as a callback function

Reply:
Done